content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,856 @@
+
+
+GOVC will also require the same VMWare vCenter details as environment variables, as you defined earlier in the **.packerenv** file. Use the following command to source the **.packerenv** file and echo one of the variables to ensure the variables are accessible on your host machine. 
+<br />
+
+```bash
+source .packerenv
+echo $PKR_VAR_vcenter_server
+```
+
+Use the following heredoc script to create a file, **.goenv**, containing the VMWare vCenter details as environment variables. The script will reuse the values sourced from the **.packerenv** file.
+<br />
+
+```bash
+cat << EOF > .goenv
+vcenter_server=$PKR_VAR_vcenter_server
+vcenter_username=$PKR_VAR_vcenter_username
+vcenter_password=$PKR_VAR_vcenter_password
+vcenter_datacenter=$PKR_VAR_vcenter_datacenter
+vcenter_datastore=$PKR_VAR_vcenter_datastore
+vcenter_resource_pool=$PKR_VAR_vcenter_resource_pool
+vcenter_folder=$PKR_VAR_vcenter_folder
+vcenter_cluster=$PKR_VAR_vcenter_cluster
+vcenter_network=$PKR_VAR_vcenter_network
+EOF
+```
+
+
+Issue the following command to clone the VM template and provision three VMs in the VMWare vCenter. The `--env-file` option in the command below will read the environment file. Then it changes to the **edge-native/vmware/clone_vm_template/** directory in the container and executes the **deploy-edge-host.sh** script to provision three VMs. 
+<br />
+
+```bash
+docker run -it --rm --env-file .goenv ghcr.io/spectrocloud/tutorials:1.0.6 sh --cpu-shares  "cd edge-native/vmware/clone_vm_template/ && ./deploy-edge-host.sh"
+```
+
+The cloning process can take up to 3-4 minutes to finish, and display the following output.
+<br />
+
+```bash coloredLines=6-6 hideClipboard
+# Sample output
+Cloned VM demo-1
+Powering on VM demo-1
+Powering on VirtualMachine:vm-9919... OK
+Getting UUID demo-1
+Edge Host ID   VM demo-1 : edge-97f2384233b498f6aa8dec90c3437c28
+```
+
+The output on the terminal will display the Edge host ID for all VMs. VMs will use this host ID to auto-register themselves with Palette. 
+<br />
+
+<InfoBox>
+
+Copy the Edge host ID for future reference and manual registration if the auto registration fails, for any reason, later in this tutorial. 
+
+</InfoBox>
+
+
+Note that the **edge-native/vmware/clone_vm_template/** directory in the container has the following files:
+- **deploy-edge-host.sh** - Provisions the VMs.
+- **delete-edge-host.sh** - Deletes the VMs.
+- **setenv.sh** - Defines the GOVC environment variables, the number of VMs, a prefix string for the VM name, and the VM template name. Most of the GOVC environment variables point to the default environment variables you have defined in the **.goenv** file. 
+
+Here is the **setenv.sh** file content for your reference; however, you do not have to modify these configuration in this tutorial. 
```

Comment:
Same as earlier, if you're gonna mention this then do it before the command that uses this file.